srcpkgs/pinebookpro-kernel/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'pinebookpro-kernel'
 pkgname=pinebookpro-kernel
-version=5.9.12
+version=5.10.1
```

Comment:
Why not .2?

Reply:
When I started this, it was at .1... I should have checked this morning :/